### PR TITLE
add amp-selector sample

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@kul3r4 @camelburrito PTAL

File: src/20_Components/amp-selector.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<!--
```
Comment (kul3r4) [inline src/20_Components/amp-selector.html]: We need to add the experiment tag here.

Comment (sebastianbenz) [inline src/20_Components/amp-selector.html]: It works and validates out of the box.

**Comment by sebastianbenz** [comment]
Added experiment flag - ptal

**Comment by kul3r4** [comment]
LGTM

**Pull request merged**